atomics/T1059.004/T1059.004.yaml

```diff
@@ -243,3 +243,14 @@ atomic_tests:
     elevation_required: false
     command: |
       uname -srm
+- name: Shell Creation using awk command
+  auto_generated_guid: 
+  description: |-
+    In awk the begin rule runs the first record without reading or interpreting it. This way a shell can be created and used to break out from restricted environments with the awk command.
+    Reference - https://gtfobins.github.io/gtfobins/awk/#shell
+  supported_platforms:
+  - linux
+  - macos
+  executor:
+    command: awk 'BEGIN {system("/bin/sh &")}'
```

Comment:
@nish221b-bs I made 2 changes to the content that you had. 
- Since this one spawns a shell and doesn't close it, `Invoke-AtomicTest` was waiting for it to close. Would be hard for users who are doing periodic automated tests. So I appended `&` in the end to make it a background job. 
- As `awk` is available in macos as well, I thought it could be reused for macos too.

Reply:
Wondering your thoughts about this before merging the PR

Reply:
Hello @cyberbuff Yeah sure, that makes sense. Thank you for the modifications. I am ok with merging the PR.

Reply:
Thank you for your contribution. Keep it up 👍